libs/api/domains/signature-collection/src/lib/guards/userAccess.guard.ts

```
@@ -54,8 +54,12 @@ export class UserAccessGuard implements CanActivate {
 
   private determineCollectionType = (context: ExecutionContext) => {
     const { body } = getRequest(context)
-    if (body && body.collectionType && isCollectionType(body.collectionType)) {
-      return body.collectionType as CollectionType
+    if (
+      body &&
+      body.variables.input.collectionType &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)